packages/itwinui-css/src/select/select.scss

```diff
@@ -39,7 +39,7 @@
 
   .iui-icon svg {
     @include menu.iui-menu-icon;
-    fill: currentColor;
+    fill: var(--iui-color-icon);
```

Comment:
I changed this to pink and it did not show up in any tests. Maybe not necessary anymore?

Reply:
confirm it in react too, then remove it

Reply:
Checked in our CSS demos, React demos, and test images: no change.

Dev tools shows it being overrode:
![Screenshot 2024-01-18 at 6 22 34 PM](https://github.com/iTwin/iTwinUI/assets/849817/47be5f2a-d413-4c77-a5ee-b184b3ad2c1d)

Removing it.